apis/hive/v1/nutanix/platform.go

```diff
@@ -0,0 +1,95 @@
+package nutanix
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Platform stores any global configuration used for Nutanix platform
+type Platform struct {
+	// PrismCentral is the endpoint (address and port) to connect to the Prism Central.
+	// This serves as the default Prism-Central.
+	PrismCentral PrismEndpoint `json:"prismCentral"`
```

Comment:
Don't you need the PC username/password for the install-config.yaml for installer? The installer IPI for Nutanix will use the PC username/password to make the prism-API calls to validate the Nutanix platform configuration in the  install-config.yaml.

Reply:
The user supply the credentials as a separate secret on hive